src/Repository/AnnouncementRepository.php

```diff
@@ -30,8 +36,23 @@ public function findToday()
       ->orderBy('a.id', 'ASC')
       ->getQuery()
       ->getResult()
-    ;
+      ;
+
+  }
 
+  /**
+   * Custom method to pull all announcements from the logged in user
+   * 
+   * @author Daniel Boling
+   */
+  public function findByUser($user)
+  {
+    return $this->createQueryBuilder('u')
+      ->andWhere('u.User = :user')
+      ->setParameter('user', $user)
+      ->getQuery()
+      ->getResult()
+      ;
```

Comment:
This isn't bad to have but if you make the change I recommended in `OverviewController.php` this won't be needed